core/src/main/java/org/apache/gravitino/connector/GenericLakehouseTable.java

```diff
@@ -47,11 +47,16 @@ protected GenericLakehouseTable internalBuild() {
       GenericLakehouseTable genericLakehouseTable = new GenericLakehouseTable();
       genericLakehouseTable.columns = this.columns;
       genericLakehouseTable.comment = this.comment;
-      genericLakehouseTable.properties =
-          ImmutableMap.<String, String>builder()
-              .putAll(this.properties)
-              .put(Table.PROPERTY_TABLE_FORMAT, this.format)
-              .buildKeepingLast();
+
+      if (format != null) {
+        genericLakehouseTable.properties =
+            ImmutableMap.<String, String>builder()
+                .putAll(this.properties)
+                .put(Table.PROPERTY_TABLE_FORMAT, this.format)
+                .buildKeepingLast();
+      } else {
```

Comment:
Should we allow the `format` property in `GenericLakehouseTable` to be null?

Reply:
No, for the current implementation, it is not allowed.

Reply:
if so, should we throw an exception in the `else` block?

Reply:
This is the temporary code, I will remove it in the follow-up PR.